dataloader/tasks_segments_parallels.py

```diff
@@ -72,12 +75,11 @@ def create_files_segments_graph() -> None:
     )
 
 
-def load_segment_data_from_menu_files(root_url: str, threads: int):
-    for language in DEFAULT_LANGS:
+def load_segment_data_from_menu_files(root_url: str, threads: int, list_of_langs: list):
```

Comment:
nitpicking: we don't need to put the type of the variable in the name. IMO `langs: list` would be enough ;)

Reply:
alright!